packages/uniforms-bridge-simple-schema-2/src/SimpleSchema2Bridge.ts

```diff
@@ -160,9 +160,12 @@ export default class SimpleSchema2Bridge extends Bridge {
     return props;
   }
 
+  makeGeneric(name?: string): string | undefined {
+    return name?.replace(/\.\d+(\.|$)/g, '.$$$1');
+  }
```

Comment:
Move it out of the bridge into a helper function - we don't want to increase the API surface here.

Reply:
Okay, moved out